pkg/chains/signing/kms/kms.go

```diff
@@ -41,6 +41,18 @@ type Signer struct {
 // NewSigner returns a configured Signer
 func NewSigner(ctx context.Context, cfg config.KMSSigner) (*Signer, error) {
 	kmsOpts := []signature.RPCOption{}
+
+	if cfg.Auth.Address != "" {
+		resp, err := http.Get(cfg.Auth.Address)
```

Comment:
Is this expected to always work without providing any http authentication?

Reply:
Yeah that's a good point 👍🏻

Reply:
do you think `net.DialTimeout()` would be a better approach?
https://pkg.go.dev/net#DialTimeout